# Generated for NaoQi method ALLogger.separator
# output a simple line separator

---


